Sources/SpeziLLMOpenAI/FunctionCalling/LLMFunctionParameterWrapper+CustomTypes.swift

```diff
@@ -18,63 +18,76 @@ extension _LLMFunctionParameterWrapper where T: AnyArray, T.Element: LLMFunction
-    public convenience init<D: StringProtocol>(
-        description: D,
+    public convenience init(
+        description: some StringProtocol,
         minItems: Int? = nil,
         maxItems: Int? = nil,
         uniqueItems: Bool? = nil
     ) {
-        self.init(schema: .init(
-            type: .array,
-            description: String(description),
-            items: .init(
-                type: T.Element.itemSchema.type,
-                properties: T.Element.itemSchema.properties,
-                pattern: T.Element.itemSchema.pattern,
-                const: T.Element.itemSchema.const,
-                enum: T.Element.itemSchema.enum,
-                multipleOf: T.Element.itemSchema.multipleOf,
-                minimum: T.Element.itemSchema.minimum,
-                maximum: T.Element.itemSchema.maximum
-            ),
-            minItems: minItems,
-            maxItems: maxItems,
-            uniqueItems: uniqueItems
-        ))
+        do {
+            let itemSchema = T.Element.itemSchema.value
+            try self.init(schema: .init(unvalidatedValue: [
+                "type": "array",
+                "description": String(description),
+                "items": [
+                    "type": itemSchema["type"],
+                    "properties": itemSchema["properties"],
+                    "pattern": itemSchema["pattern"],
+                    "const": itemSchema["const"],
+                    "enum": itemSchema["enum"],
+                    "multipleOf": itemSchema["multipleOf"],
+                    "minimum": itemSchema["minimum"],
+                    "maximum": itemSchema["maximum"]
+                ].compactMapValues { $0 },
+                "minItems": minItems as Any?,
+                "maxItems": maxItems as Any?,
+                "uniqueItems": uniqueItems as Any?
+            ].compactMapValues { $0 }))
+        } catch {
+            logger.error("Couldn't create FunctionParameterWrapper+CustomType \(error)")
+            self.init(description: "")
+        }
     }
 }
 
-extension _LLMFunctionParameterWrapper where T: AnyOptional, T.Wrapped: AnyArray, T.Wrapped.Element: LLMFunctionParameterArrayElement {
+extension _LLMFunctionParameterWrapper where T: AnyOptional, T.Wrapped: AnyArray,
+                                             T.Wrapped.Element: LLMFunctionParameterArrayElement {
     /// Declares an optional ``LLMFunctionParameterArrayElement``-based (custom type) ``LLMFunction/Parameter`` `array`.
     ///
     /// - Parameters:
     ///    - description: Describes the purpose of the parameter, used by the LLM to grasp the purpose of the parameter.
     ///    - minItems: Defines the minimum amount of values in the `array`.
     ///    - maxItems: Defines the maximum amount of values in the `array`.
     ///    - uniqueItems: Specifies if all `array` elements need to be unique.
-    public convenience init<D: StringProtocol>(
-        description: D,
+    convenience init(
```

Comment:
Why did you remove the `public` accessor?

Reply:
Addresssed in 22a97319fef2e6ec0e006b1b1523df78bc81addd